src/locus_loader.erl

```diff
@@ -335,6 +337,10 @@ validate_loader_opts(MixedOpts, FetcherOpts) ->
                   orelse error({badopt, Opt});
               (no_cache) ->
                   true;
+              ({database_cache_file, File}) ->
+                  % Ensure directory exists
+                  Dirname = filename:dirname(File),
+                  filelib:is_dir(Dirname);
```

Comment:
```suggestion
                  filelib:is_dir(Dirname)
                  orelse error({badopt, Opt});
```

So that it doesn't get placed under `OtherOpts`:
https://github.com/g-andrade/locus/blob/3037d490254634972bb26c234b450110173c3abd/src/locus_loader.erl#L343-L344

(Although the end result would be the same - I think - as `locus_database` wouldn't recognize it as an option either and fail appropriately :thinking: 
https://github.com/g-andrade/locus/blob/3037d490254634972bb26c234b450110173c3abd/src/locus_database.erl#L338-L339 

...but we might as well initiate the error flow here.)

Reply:
(Oh man, why did I ever consider throwing exceptions from within a list iteration callback a good idea...)